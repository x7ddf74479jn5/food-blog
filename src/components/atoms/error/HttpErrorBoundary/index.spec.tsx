import renderer from "react-test-renderer";

import { HttpErrorBoundary } from ".";

describe("components/atoms/error/HttpErrorBoundary", () => {
  it("snapshot", () => {
    const tree = renderer.create(<HttpErrorBoundary>children</HttpErrorBoundary>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
