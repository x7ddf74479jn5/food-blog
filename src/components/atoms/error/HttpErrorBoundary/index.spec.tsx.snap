// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/error/HttpErrorBoundary snapshot 1`] = `"children"`;
